content/api/commands/intercept.md

```diff
@@ -22,136 +22,64 @@ Unlike [cy.route()](/api/commands/route), `cy.intercept()`:
 - does not require calling [cy.server()](/api/commands/server) before use - in fact, `cy.server()` does not influence `cy.intercept()` at all.
 - does not have method set to `GET` by default, but intercepts `*` methods.
 
-## Usage
+## Syntax
 
-```ts
-cy.intercept(url, routeHandler?)
-cy.intercept(method, url, routeHandler?)
-cy.intercept(routeMatcher, routeHandler?)
+### Route-matching Only (No Stubbing or Intercepting)
+```js
+cy.intercept(url)
+cy.intercept(method, url)
+cy.intercept(routeMatcher)
+cy.intercept(url, routeMatcher)
+cy.intercept(method, url, routeMatcher)
+```
+
+### Route-matching With Stubbing and Intercepting
+```js
+cy.intercept(url, routeHandler)
+cy.intercept(method, url, routeHandler)
+cy.intercept(routeMatcher, routeHandler)
 cy.intercept(url, routeMatcher, routeHandler)
+cy.intercept(method, url, routeMatcher, routeHandler)
 ```
 
 **Note:** all intercepts are automatically cleared before every test.
 
-### Arguments
-
-#### **<Icon name="angle-right"></Icon> url** **_(`string | RegExp`)_**
+### Usage
 
-Specify the URL to match. See the examples for [Matching URL](#Matching-URL) to see how URLs are matched.
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
 
-```ts
-cy.intercept('http://example.com/widgets')
-cy.intercept('http://example.com/widgets', { fixture: 'widgets.json' })
+```js
+cy.intercept('/users/**')
 ```
+### Arguments
 
-#### **<Icon name="angle-right"></Icon> method** **_(`string`)_**
+**<Icon name="angle-right"></Icon> url** **_(String, Glob, RegExp)_**
 
-Specify the HTTP method to match on.
+Specify the URL to match. See [Matching URL](#Matching-URL) for examples.
 
-```ts
-cy.intercept('POST', 'http://example.com/widgets', {
-  statusCode: 200,
-  body: 'it worked!',
-})
-```
+**<Icon name="angle-right"></Icon> method** **_(String)_**
 
-#### **<Icon name="angle-right"></Icon> routeMatcher** **_(`RouteMatcher`)_**
+Specify the [HTTP request method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods) (`GET`, `POST`, `PUT`, etc.) to match on.
 
-`routeMatcher` is an object used to match which incoming HTTP requests will be handled by this route.
+**<Icon name="angle-right"></Icon> routeMatcher** **_(`RouteMatcher`)_**
```

Comment:
Implied standards for an argument in an Arguments section:
* not in a header, but bold and camelCase
* prefixed with an angle-right icon
* types are italicized and PascalCase
* types are wrapped in parens and separated by commas

Reply:
make them a header, remove bold?